backend/src/main/scala/cromwell/backend/BackendLifecycleActor.scala

```diff
@@ -63,3 +53,19 @@ trait BackendLifecycleActor extends Actor {
     }
   }
 }
+
+trait BackendWorkflowLifecycleActor extends BackendLifecycleActor {
+  /**
+    * The workflow descriptor for the workflow in which this Backend is being used
+    */
+  protected def workflowDescriptor: BackendWorkflowDescriptor
+
+  /**
+    * The subset of calls which this backend will be expected to run
+    */
+  protected def calls: Seq[Call]
```

Comment:
Why do we need it separately? Can't it be accessed from the namespace? (`namespace.workflow.calls`)